### ISSUE TITLE
Setting Python Path on Win 10

### ISSUE THREAD

**Issue by alperyazar** [TerosTechnology/vscode-terosHDL#91]
Hi there,

Thanks for this fantastic plugin. I am experimenting with it on Win 10. I can't use *Dependencies Viewer* because it can't find Python. I have installed Python 3.9.2 and it's on the path, i.e. I can run `python` on cmd or PowerShell. I also tried to manually enter the path in extension settings but i didn't work. If I have to enter the path manually, how should I enter the python path, `C:\Python\python.exe` or `C:\\Python\\python.exe` (with escaping), etc.? 

Thank you.

**Comment by alperyazar** [comment]
My mistake, `C:\\Python\\python.exe` works. I had typo in the settings and noticed after opening the issue. It still doesn't work even `python` is in the path and explicit set is required but double `\` does the trick.

The issue can be closed if it is OK for developers.

**Comment by smgl9** [comment]
@alperyazar Thanks for the feedback and thanks for the workaround!
I left the issue open because the "trick" shouldn't be necessary. It happens in windows only.

**Comment by alperyazar** [comment]
@smgl9 I agree, thank you!

**Comment by monk31** [comment]
hi ,
i have the same problem with windows10, my version is 0.1.4
the rule is double quote + python.exe at the end of the string ?

**Comment by alperyazar** [comment]
> 
> 
> hi ,
> i have the same problem with windows10, my version is 0.1.4
> the rule is double quote + python.exe at the end of the string ?

Hi,
Sorry for the late response. I don't have access to the machine right now but as far as I remember, yes, like ``C:\\Python\\python.exe``

**Comment by Hqf2013** [comment]
I also encountered the same problem. The python setting is invalid under win10.

**Comment by qarlosalberto** [comment]
We are working in the new version to fix the Windows issues.

**Comment by ziyangd** [comment]
hi I have the same issue under win10, double / does not work.

**Comment by qarlosalberto** [comment]
Have you triend double \ ?

**Comment by ziyangd** [comment]
> 
yes I tried double \ and python is in my path. So where could be the problem?  
![teroshdl](https://user-images.githubusercontent.com/70007046/146879443-49188ec2-75d7-469a-8c05-d82fabdaeb32.jpg)
![python](https://user-images.githubusercontent.com/70007046/146879452-9cff7db9-d580-4903-83dd-c9f12a5bc52f.jpg)

**Comment by mlogelect** [comment]
Same problem here, i have python in my path and i have installed everything, but this is what i get
![rgrg](https://user-images.githubusercontent.com/95703647/153748728-829a9fc5-3294-425c-97f4-966a38d6be1c.png)

**Comment by qarlosalberto** [comment]
@mlogelect open from cmd the python console. Type "import vunit", does it work?

**Comment by mlogelect** [comment]
I tried and couldn't see the vunit. So i re-installed python3 and searched the path in command line again. This time it shows me a different path than before. Now everything works, thank you!

**Comment by qarlosalberto** [comment]
Cool! I don't use Windows, so it's more difficult for me test it...
